src/main/java/pl/learning/bankAccount/Account.java

```diff
@@ -16,46 +16,46 @@ public Account(String number, double balance, String name, String email, String
         this.phoneNumber = phoneNumber;
     }
 
-    public void setNumber(String number) {
-        this.number = number;
-    }
-
-    public void setBalance(double balance) {
-        this.balance = balance;
-    }
-
-    public void setName(String name) {
-        this.name = name;
-    }
-
-    public void setEmail(String email) {
-        this.email = email;
-    }
-
-    public void setPhoneNumber(String phoneNumber) {
-        this.phoneNumber = phoneNumber;
-    }
-
     public String getNumber() {
```

Comment:
Getters and setters of the same fields should be grouped together, thats why I moved setters under the getters

Reply:
Thanks for tips. We need to talk about setters, getters, this, super some day ;D